webrtc.html

```diff
@@ -3472,6 +3485,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               framerate, <code>degradationPreference</code> indicates what should be prefered.
             </p>
           </dd>
+
+          <dt>DOMString rid</dt>
+          <dd>
+            <p> If set, this RTP encoding will be sent with the RID header extension as
+              defined by [[!RTCWEB-JSEP]].
```

Comment:
Why does this need to be set, can we not simply require that browsers use RID?  Having the browser choose the RID will work much better than allowing the application pick something.